src/Cashier.php

```diff
@@ -19,14 +19,14 @@ class Cashier
      *
      * @var string
      */
-    const VERSION = '15.6.4';
+    const VERSION = '16.0.0';
 
     /**
      * The Stripe API version.
      *
      * @var string
      */
-    const STRIPE_VERSION = '2023-10-16';
+    const STRIPE_VERSION = '2025-03-31.basil';
```

Comment:
When this is eventually merged and a new cashier version is tagged, it might be best to target the latest version of basil.

The latest basil version is now `2025-06-30.basil`

https://docs.stripe.com/changelog/basil#2025-06-30.basil

Reply:
Shouldn't we rely on https://github.com/stripe/stripe-php/blob/v17.4.0/API_VERSION value?

Reply:
Also, what will happen when `stripe/stripe-php` release a minor version using an updated API version? Do we need to lock the dependencies to patch release only?